### PR TITLE
Use println instead of warn to avoid missing deprecated message

### PR DESCRIPTION
**Pull request by b1tg**
Signed-off-by: b1tg <b1tg@protonmail.ch>

Fix https://github.com/mozilla/sccache/issues/1484

### CODE REVIEW

File: src/config.rs [hunk 1]
```diff
@@ -537,8 +537,8 @@ fn config_from_env() -> Result<EnvConfig> {
 
 
         if env::var("SCCACHE_GCS_OAUTH_URL").is_ok() {
-            warn!("SCCACHE_GCS_OAUTH_URL has been deprecated");
-            warn!("if you intend to use vm metadata for auth, please set correct service account intead");
+            println!("SCCACHE_GCS_OAUTH_URL has been deprecated");
```
Comment (glandium) [inline src/config.rs:540]: this should probably use eprintln

Comment (b1tg) [reply src/config.rs]: fixed

**Review by Xuanwo — APPROVED** [review]

**Comment by drahnr** [comment]
Could we possibly print _all_ warning messages instead? I don't know if that'd be excessive. It have a hunch we have more places where a warning to the user would be desirable.

**Comment by sylvestre** [comment]
Let's start here and follow up in other pr if needed

**Pull request merged**